Python/Product/TestAdapter.Executor/PythonFiles/testing_tools/run_adapter.py

```diff
@@ -4,9 +4,11 @@
 # Replace the "." entry.
 import os.path
 import sys
-sys.path[0] = os.path.dirname(
-    os.path.dirname(
-        os.path.abspath(__file__)))
+
```

Comment:
is this part of their changes? do you know what effect this has?

Reply:
This was change Kim-Adeline Miguel Fix overwriting of cwd in test discovery path list (#6713)
https://github.com/microsoft/vscode-python/pull/6713

i'll ask her more about it

Reply:
https://github.com/microsoft/vscode-python/issues/6417